packages/network-query/leaderboard.codegen.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020-2022 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { CodegenConfig } from '@graphql-codegen/cli';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const config: CodegenConfig = {
```

Comment:
May I confirm we confirm to include `leaderboard`?

Reply:
it‘s for fetch topindexer